### DISCUSSION TITLE
Pending GitHub Sponsors Application

### DISCUSSION THREAD

**Discussion by bensonoyugi** [community/community#147323] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi GitHub Support Team,
I submitted my GitHub Sponsors application on September and it is still pending with absolutely no updates. I am not sure what is wrong as I know colleagues who applied much later than me and had theirs accepted in just a few days. Could you please assist me with this issue? 

Best regards,
bensonoyugi

**Comment by mecodeatlas** [comment]
Hi @bensonoyugi! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by bensonoyugi** [reply]
Thank you.
